public/stylesheets/site/2.0/02-elements.css

```diff
@@ -129,9 +129,15 @@ blockquote p:last-child {
 
 kbd, tt, code, var, pre, samp {
   font: normal 0.857em 'Monaco', 'Consolas', Courier, monospace;
+  font-variant-ligatures: none;
   margin: auto;
 }
 
+code.big {
```

Comment:
the `big` class clashes with a native HTML tag and I don't think we name classes this way. I think this might need the eyes of sarken? The text also feels too big